### PR TITLE
Announce IstioDay at KubeCon Europe 2023

### PR DESCRIPTION
**Pull request by kfaseela**
Signed-off-by: Faseela K <faseela.k@est.tech>

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2023/istioday-kubecon-eu/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Join us for IstioDay at KubeCon Europe 2023!"
+description: The first ever IstioDay at KubeCon will take place in April.
+publishdate: 2023-01-26
+attribution: "Istio Steering Committee"
+keywords: [IstioCon,Istio,conference]
+---
+
+IstioDay Europe 2023, set for April 18, will be the first ever conference for Istio, the industry’s [most popular service mesh](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf), collocated with the KubeCon. Istio, which is currently [a graduating project at CNCF](https://www.cncf.io/projects/istio/) will mark its presence at the KubeCon in 2023, connecting community members across the globe with Istio’s ecosystem. Visit the [conference website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/istio-day/) for all the information related to the event.
```
Comment (therealmitchconnors) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:9]:
```suggestion
IstioDay Europe 2023, set for April 18, will be the first ever in-person conference for Istio, the industry’s [most popular service mesh](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf), collocated with KubeCon Europe. Istio, which is currently [an incubating project at CNCF](https://www.cncf.io/projects/istio/) will celebrate its sixth birthday at IstioDay, connecting community members across the globe with Istio’s ecosystem. Visit the [conference website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/istio-day/) for all the information related to the event.
```

Comment (ericvn) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:9]:
```suggestion
IstioDay Europe 2023, set for April 18, will be the first ever conference for Istio, the industry’s [most popular service mesh](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf), collocated with KubeCon. Istio, which is currently [a graduating project at CNCF](https://www.cncf.io/projects/istio/) will mark its presence at the KubeCon in 2023, connecting community members across the globe with Istio’s ecosystem. Visit the [conference website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/istio-day/) for all the information related to the event.
```

Comment (ericvn) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:9]: Have we mentioned we are trying to graduate at this point?

File: content/en/blog/2023/istioday-kubecon-eu/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+{{< image width="75%"
+    link="./istioday-eu-logo.png"
+    alt="IstioDay logo"
+    >}}
+
+IstioDay is the perfect opportunity to showcase the lessons learned from running Istio in production, hands-on experiences from the Istio community, and will feature maintainers from across the Istio ecosystem. IstioDay would focus on sharing real world examples, case studies, and success stories that can inspire newcomers to use Istio in production. The content will range from introductory to advanced levels, split into four main topic tracks:
```
Comment (therealmitchconnors) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:16]:
```suggestion
IstioDay is the perfect opportunity to showcase lessons learned from running Istio in production, hands-on experiences from the Istio community, and will feature maintainers from across the Istio ecosystem. IstioDay will include real world examples, case studies, and success stories that can inspire newcomers to use Istio in production. The content will range from introductory to advanced levels, split into four main topic tracks:
```

File: content/en/blog/2023/istioday-kubecon-eu/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+* **New Features:** New standards, new extensions and how to address problems that are interesting to tackle
+* **Case Studies:** Observability, traceability, and other things built on top of Istio.
+* **Istio Recipes:** How Istio works, with deep-dives into performance, cost, and multi-cloud environments.
+* **Project Updates:** The evolution of Istio, and the latest updates from the project maintainers.
+
+At this time, we encourage Istio users, developers, partners, and advocates to [submit a session proposal through the conference’s CFP portal](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/cfp-colocated-events/). IstioDay offers a mix of technical talks, lightning talks, workshops, and roadmap sessions. Choose from the following formats to submit a session proposal for IstioDay:
```
Comment (therealmitchconnors) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:23]:
```suggestion
At this time, we encourage Istio users, developers, partners, and advocates to [submit a session proposal through the conference’s CFP portal](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/cfp-colocated-events/), which is scheduled to close on Feb 12. Choose from the following formats to submit a session proposal for IstioDay:
```

File: content/en/blog/2023/istioday-kubecon-eu/index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+* **Presentation:** 25 minutes, 1 or 2 speaker(s) presenting a topic
+* **Panel Discussion:** 35 minutes of discussion among 3 to 5 speakers
+* **Lighting Talk:** A brief 5-minute presentation, maximum of 1 speaker
+
+At KubeCon Europe, Istio community also has an interactive social hour to take the load off and mesh with the Istio community, vendors, and maintainers. IstioDay is one of the `KubeCon + CloudNativeCon Europe CNCF-hosted Co-located Events`. In-person attendees will now have the option to register for an `All-Access In-Person KubeCon + CloudNativeCon pass` that will include entry to all `CNCF-hosted co-located events + KubeCon + CloudNativeCon`.
```
Comment (therealmitchconnors) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:29]: Can we provide a link to the social?

File: content/en/blog/2023/istioday-kubecon-eu/index.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+{{< image width="75%"
+    link="./istioday-eu-logo.png"
+    alt="IstioDay logo"
+    >}}
+
+IstioDay is the perfect opportunity to showcase the lessons learned from running Istio in production, hands-on experiences from the Istio community, and will feature maintainers from across the Istio ecosystem. IstioDay would focus on sharing real world examples, case studies, and success stories that can inspire newcomers to use Istio in production. The content will range from introductory to advanced levels, split into four main topic tracks:
+
+* **New Features:** New standards, new extensions and how to address problems that are interesting to tackle
```
Comment (ericvn) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:18]: Needs a period at the end.  Don't like the `the are interesting to tackle`.

File: content/en/blog/2023/istioday-kubecon-eu/index.md (hunk 4 above)
Comment (ericvn) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:29]:
```suggestion
At KubeCon Europe, the Istio community also has an interactive social hour to take the load off and mesh with the Istio community, vendors, and maintainers. IstioDay is one of the `KubeCon + CloudNativeCon Europe CNCF-hosted Co-located Events`. In-person attendees will now have the option to register for an `All-Access In-Person KubeCon + CloudNativeCon pass` that will include entry to all `CNCF-hosted co-located events + KubeCon + CloudNativeCon`.
```

**Review by therealmitchconnors — CHANGES_REQUESTED** [review]

File: content/en/blog/2023/istioday-kubecon-eu/index.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+* **Presentation:** 25 minutes, 1 or 2 speaker(s) presenting a topic
+* **Panel Discussion:** 35 minutes of discussion among 3 to 5 speakers
+* **Lighting Talk:** A brief 5-minute presentation, maximum of 1 speaker
+
+At KubeCon Europe, Istio community also has an interactive social hour to take the load off and mesh with the Istio community, vendors, and maintainers. IstioDay is one of the `KubeCon + CloudNativeCon Europe CNCF-hosted Co-located Events`. In-person attendees will now have the option to register for an `All-Access In-Person KubeCon + CloudNativeCon pass` that will include entry to all `CNCF-hosted co-located events + KubeCon + CloudNativeCon`.
+
+The most exciting factor this time in KubeCon is that the accepted co-located event speakers will receive a complimentary `All-Access In-Person ticket`, providing free access to all `CNCF-hosted co-located events` and `KubeCon + CloudNativeCon`.
```
Comment (ericvn) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:31]:
```suggestion
The most exciting factor this KubeCon is that the accepted co-located event speakers will receive a complimentary `All-Access In-Person ticket`, providing free access to all `CNCF-hosted co-located events` and `KubeCon + CloudNativeCon`.
```

**Review by ericvn — COMMENTED** [review]
A quick first pass.

File: content/en/blog/2023/istioday-kubecon-eu/index.md (hunk 4 above)
Comment (kfaseela) [reply content/en/blog/2023/istioday-kubecon-eu/index.md]: I think we do not have a link at the moment. Should I remove this mention?

File: content/en/blog/2023/istioday-kubecon-eu/index.md (hunk 1 above)
Comment (kfaseela) [reply content/en/blog/2023/istioday-kubecon-eu/index.md]: No, it was a mistake. fixed.

File: content/en/blog/2023/istioday-kubecon-eu/index.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+* **New Features:** New standards, new extensions and how to address problems that are interesting to tackle.
+* **Case Studies:** Observability, traceability, and other things built on top of Istio.
+* **Istio Recipes:** How Istio works, with deep-dives into performance, cost, and multi-cloud environments.
+* **Project Updates:** The evolution of Istio, and the latest updates from the project maintainers.
+
+At this time, we encourage Istio users, developers, partners, and advocates to [submit a session proposal through the conference’s CFP portal](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/cfp-colocated-events/), which is scheduled to close on Feb 12. Choose from the following formats to submit a session proposal for IstioDay:
+
+* **Presentation:** 25 minutes, 1 or 2 speaker(s) presenting a topic
+* **Panel Discussion:** 35 minutes of discussion among 3 to 5 speakers
+* **Lighting Talk:** A brief 5-minute presentation, maximum of 1 speaker
```
Comment (frankbu) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:27]:
```suggestion
* **Lightning Talk:** A brief 5-minute presentation, maximum of 1 speaker
```

File: content/en/blog/2023/istioday-kubecon-eu/index.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+IstioDay is one of the `KubeCon + CloudNativeCon Europe CNCF-hosted Co-located Events`. In-person attendees will now have the option to register for an `All-Access In-Person KubeCon + CloudNativeCon pass` that will include entry to all `CNCF-hosted co-located events + KubeCon + CloudNativeCon`.
+
+The most exciting factor this KubeCon is that the accepted co-located event speakers will receive a complimentary `All-Access In-Person ticket`, providing free access to all `CNCF-hosted co-located events` and `KubeCon + CloudNativeCon`.
+
+Do not forget to take a look at [the dates to remember section](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/cfp-colocated-events/#general-info), and do not miss the deadline. Stay tuned to hear more about the event, and we hope you can join us at IstioDay at KubeCon Amsterdam, April 18, 2023!
```
Comment (craigbox) [inline content/en/blog/2023/istioday-kubecon-eu/index.md:33]:
```suggestion
---
title: "Join us for Istio Day at KubeCon Europe 2023!"
description: The call for session proposals is now open.
publishdate: 2023-01-27
attribution: "Istio Steering Committee"
keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
---

Istio is sailing up the canals this April! We are delighted to announce [Istio Day Europe 2023](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/istio-day/), a "Day 0" event co-located with [KubeCon + CloudNativeCon Europe 2023](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe).

{{< image width="75%"
    link="./istioday-eu-logo.png"
    alt="Istio Day Europe, 18 April 2023, RAI, Amsterdam, The Netherlands. #istioday"
    >}}

Istio Day is the perfect opportunity to meet the Istio maintainers and contributors in person, and hear from users why Istio is constantly ranked the #1 service mesh in production.

## Submit a talk

We now encourage Istio users, developers, partners, and advocates to [submit a session proposal](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/cfp-colocated-events/) through the CNCF event portal, which is open until **February 12**.

We want to see real world examples, case studies, and success stories that can inspire newcomers to use Istio in production. The content will cover introductory to advanced levels, split into four main topic tracks:

* **New Features:** What have you been working on that the community should know about?
* **Case Studies:** How have you built a platform or service on top of Istio?
* **Istio Recipes:** How you can solve a specific business problem using Istio.
* **Project Updates:** The evolution of Istio, and the latest updates from the project maintainers.

You can pick one of these formats to submit a session proposal:

* **Presentation:** 25 minutes, 1 or 2 speaker(s) presenting a topic
* **Panel Discussion:** 35 minutes of discussion among 3 to 5 speakers
* **Lighting Talk:** A brief 5-minute presentation, maximum of 1 speaker

Accepted speakers will receive a complimentary All-Access In-Person ticket for *all four days* of KubeCon + CloudNativeCon.

## Sponsor the event

Do you want to put your product or service in front of the most discerning Cloud Native users: those who demand *25% more conference* then the crowd? Check out [page 11 of the CNCF events prospectus](https://events.linuxfoundation.org/wp-content/uploads/2023/01/sponsor-cncf-2023-012523.pdf) to learn more.

## Register to attend

[Istio Day is a CNCF-hosted co-located event](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/istio-day/) on 18 April 2023. KubeCon + CloudNativeCon Europe in-person attendees now have the option to [buy an All-Access ticket](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/), which includes entry to all the Day 0 events, as well as the main three days of the conference.  You must be attending KubeCon to attend Istio Day, but virtual registration options are available, and the recordings will be posted to YouTube soon after the event.

For those of you who can't make it, keep your eyes peeled for announcements of IstioCon 2023 and Istio Day North America later this year.

Stay tuned to hear more about the event, and we hope you can join us at Istio Day Europe!

```

File: .spelling [hunk 9]
```diff
@@ -527,6 +527,7 @@ istio.io.
 IstioBirthday
 IstioCon
 istioctl
+IstioDay
```
[the remark this answers is not in the document]
Comment (craigbox) [reply .spelling:530]: You also need to add `CloudNativeCon` after `cli` and `CNCF-hosted` after `cluster2`.

**Review by craigbox — CHANGES_REQUESTED** [review]

**Comment by craigbox** [comment]
https://deploy-preview-12528--preliminary-istio.netlify.app/latest/blog/2023/istioday-kubecon-eu/ LGTM

**Review by craigbox — APPROVED** [review]

**Pull request merged**